releasenotes/notes/service-merge.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: bug-fix
+area: traffic-management
+issue:
+- 50478
+releaseNotes:
+- |
+  **Fixed** merge services ports should also compare the port name besides port number, otherwise the the ClusterLoadAssignments could be messed.
```

Comment:
```suggestion
  **Fixed** merge services ports should also compare the port name besides port number, otherwise the the ClusterLoadAssignments could be incorrect.
```